style.css

```
@@ -0,0 +1,1183 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+ul {
+  list-style: none;
+}
+
+body {
+  color: #555;
+  margin: 0 auto;
+  font-size: 14px;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  line-height: 1.5;
+  min-height: 100vh;
+}
+
+.clearfix::before,
+.clearfix::after {
+  content: "";
+  display: table;
+}
+
+.clearfix::before {
+  clear: both;
+}
+
+.clearfix::after {
+  clear: both;
+}
+
+.clearfix {
+  clear: both;
+  *zoom: 1;
+}
+
+.hidden {
+  display: none;
+}
+
+/************************
+* NAVIGATION BAR STYLES *
+*************************/
+
+.nav {
+  width: 100%;
+  position: sticky;
+  top: -7%;
+  z-index: 99999;
```

change this before to merge, is causing sticky nav getting hide.
```suggestion
  top: 0;
  z-index: 99999;
```